src/components/userLoginScreen.js

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect } from 'react';
+import Backend from './backend'
+import customData from '../database/data.json'
+
+
+const UserLoginScreen = (props) => {
+
+const userlist = customData
+
+const [loginInput,setloginInput]=useState('')
+const [passwordInput,setPasswordInput]=useState('')
+const [hiddenPassword,setHiddenPassword]=useState('')
+const [hidePassword,setHidePassword]=useState(false)
+const [submitted,setSubmitted]=useState(false)
+const [error,setError] = useState(false)
+const [timerid,setTimerid]=useState(0)
+const [timerMiliseconds,setTimerMiliseconds]=useState(1000)
+
+
+
+const changeLoginHandler = (e) => {
+  setloginInput(e.target.value)
+  setError(false)
+}
+
+const changePasswordHandler = (e) => {
+//using keyPress instead of onChange to hide password with delay
+  switch (e.which){ 
+
+  case 13:
+    //return carriage
+    //console.log(e.which)
+    break
+
+  default:
+
+    let currentValue= passwordInput
+
+    //console.log(e.which)
+
+    let lastchar = String.fromCharCode(e.which)
+      currentValue+=lastchar
+        
+        setPasswordInput(currentValue)
+
+        securePassword(currentValue)
+
+        setError(false)
+  
+  }
+}
+
+
+const deleteChar = (e) => {
+  //backspace delete ascii '8'
+  if(e.which==8){
+    let currentValue= passwordInput
+        currentValue = currentValue.substring(0, currentValue.length - 1);
+    
+        setPasswordInput(currentValue)
+        securePassword(currentValue)
+        setError(false)
+
+  }
+  
+}
+const resetSubmission = () => {
+    
+      setSubmitted(false)
+
+}
+
+const handleError = (e) => {
+     
+      setError(e)
+}
+
+
+const Errormessage = () => {
+let message=''
+
+    if(error){
+      
+      message = 'Wrong password or login'
+      
+    }
+      
+    return message
+      
+    }
+
+const Back = () =>{
+  return(
+    <Backend  login={loginInput} 
+              password={passwordInput} 
+              reset={resetSubmission}
+              submitted={submitted}
+              error={(e)=>handleError(e)}
+              loginChange={props.loginChange}
+              
+              />
+
```

Comment:
I see too many spaces in between closing blocks
You should start using a linter